docs/fileio_usage.rst

```diff
@@ -0,0 +1,62 @@
+=====
+Usage
+=====
+
+The easiest way to use :py:mod:`orsopy.fileio` (the module of :py:mod:`orsopy` that includes file reading and writing) to produce **metadata-rich .ort reduced reflectometry files** involves integrating this into your data reduction workflow.
+Early in the workflow, the :py:mod:`orsopy.fileio` should be imported and an empty :code:`orsopy.fileio.orso.Orso` header object (here we also import :py:mod:`numpy` which will be used later). 
+
+.. code-block:: python 
+
+    import numpy as np
+    from orsopy.fileio.orso import Orso
```

Comment:
We should make use of the existing namespace in fileio, where we collect all necessary classes already. So I would propose `from orsopy import fileio` or `from orsopy.fileio import */Orso,... `